source/jquery.flot.js

```diff
@@ -169,6 +169,7 @@ Licensed under the MIT license.
                         show: false,
                         lineWidth: 2, // in pixels
                         barWidth: 1, // in units of the x axis
+                        absoluteBarWidth: false,
```

Comment:
We should provide documentation as to how this property works. Additionally, we need to change the documentation of 'barWidth' as "in units of the x axis" is not correct. Basically, I think absoluteBarWidth should say something to the effect of "when true, the value of barWidth is considered to be in units of the x axis."

Reply:
I've update the documentation. There are still build failures that need to be addressed. Are these failures concerning, being that I am changing some existing functionality?